organizing_hub/urls.py

```diff
@@ -148,6 +149,11 @@
                         CallCampaignDetailView.as_view(),
                         name='organizing-hub-call-campaign-detail'
                     ),
+                    url(
+                        r'^(?P<uuid>[0-9a-f-]+)/update/',
```

Comment:
There is now a parent level /uuid/ url and anything under that path should be nested inside that url declaration. see call view and detail view for example. Generally don't want to redeclare url paths if we can avoid repeating ourselves.

Reply:
👍